ai_presenter/ai_presenter.py

```diff
@@ -31,3 +34,27 @@ def run(self):
         #   chr_db
         voiceai = self.generator.get_voice()
         voiceai.generate(text_ai_file, voice_ai_file)
+
+    def __clear_voices(self):
+        # file = open("test_clear_voices.txt", "w")
+
+        config = self.database.get_config()
+        key = config.get_ai_config().get_elevenlabs_api_key()
+
+        set_api_key(key)
+        voices = Voices.from_api()
```

Comment:
I'm wondering if there is a VoiceDelete API call in python, if not, this works.

But most importantly, there needs to be some type of "tag" that can be added, so that this program doesn't delete ALL voices, just the ones it created for this specific run.

For example, when it creates a VoiceDesign, it can name it `f{tag}-{actor.name}`, where `{tag}` is a variable it can use from the config file or generated or default